### PR TITLE
IMAGEADD: Mech Weapon

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Part of topic https://discord.com/channels/145533722026967040/1188998198635806750/1188998198635806750
Changes the sprite of most power weapons(and generators) to improved and/or animated versions.

## Why It's Good For The Game
Seeing the same sprites on most popular weapons is simply annoying, so I propose to add to replace the default "laser" qualitative (in my opinion) and beautiful sprites. Thus, the immolator now emits flames, the X-ray gun has a strange but similar image to its handheld counterpart, the disabler blinks, and generators no longer look like a boring box with wires.

Weapons now look like weapons, unique and merciless.

The robo rocket launcher now has a smaller sprite for 6 charges, which is why I lost 2 ammo rockets. In turn, Mauler's rocket launcher has increased its ammo by 1 to match the sprite of 9 charges. I highly doubt that these changes will critically affect the popularity of the rocket launcher, but now it looks authentic.

## Images of changes
image

## Testing
code write, game launch, all work and correct 

## Changelog
:cl:
imageadd: Mechs' energy weapons respright. Small nerf of rocket launcher from robo to feat new sprite.
/:cl:

### CODE REVIEW

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 1]
```diff
@@ -181,20 +181,20 @@
 
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/taser
+	equip_cooldown = 0.8 SECONDS
 	name = "\improper PBT \"Pacifier\" Mounted Taser"
 	icon_state = "mecha_taser"
 	origin_tech = "combat=3"
 	energy_drain = 20
-	equip_cooldown = 8
```
Comment (Henri215) [inline code/game/mecha/equipment/weapons/weapons.dm:188]: I personally would leave this var around this place instead of coming before the name

Comment (ErdGinalD) [reply code/game/mecha/equipment/weapons/weapons.dm:188]: half of the guns have a cooldown before and the other half after, I thought I'd just lump it all together.

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 2]
```diff
@@ -113,21 +113,21 @@
 	return ..()
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/tesla
-	equip_cooldown = 35
+	equip_cooldown = 3.5 SECONDS
 	name = "\improper P-X Tesla Cannon"
 	desc = "A weapon for combat exosuits. Fires bolts of electricity similar to the experimental tesla engine"
```
Comment (Henri215) [inline code/game/mecha/equipment/weapons/weapons.dm:118]:
```suggestion
	desc = "A weapon for combat exosuits. Fires bolts of electricity similar to the experimental tesla engine."
```
While we are here..

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me, another reviewer can decide where the var should be positioned.

**Review by DGamerL — COMMENTED** [review]
Github is being annoying and not letting me review this properly, but please move the `equip_cooldown` variable to under `origin_tech`

**Comment by ErdGinalD** [comment]
> Github is being annoying and not letting me review this properly, but please move the `equip_cooldown` variable to under `origin_tech`

if you say so too... okay, tomorrow will be done, after work

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**